test-cases/features/repair-based-operations/longevity-cdc-100gb-4h-Decommission.yaml

```diff
@@ -0,0 +1,21 @@
+test_duration: 260
+
+stress_cmd: [ "cassandra-stress user no-warmup profile=/tmp/cdc_profile.yaml ops'(insert=2,read1=1,update_number=1,update_name=1,delete1=1)' cl=QUORUM duration=200m -port jmx=6868 -mode cql3 native -rate threads=100",
```

Comment:
Dor mentioned in the past we should not make CDC a dependence for repair based node operations since nobody uses cdc at the moment.

Reply:
When you ready, you can choose when to run it...
Wanted to have it for the laster phase.